docs/operating-scylla/nodetool-commands/removenode.rst

```diff
@@ -15,6 +9,14 @@ See :doc:`Remove a Node from a Scylla Cluster </operating-scylla/procedures/clus
     See :doc:`Remove a Node from a Scylla Cluster </operating-scylla/procedures/cluster-management/remove-node>` for more information.
 
 
+This command allows you to remove a node from the cluster when the status of the node is Down Normal (DN) and all attempts to restore the node have failed.
+
+The node you are removing, as well as the :ref:`ignored dead nodes <removenode-ignore-dead-nodes>`, 
+are banned from the cluster at the beginning of the procedure. As a result, you 
+won't be able to bring them back, even if the ``removenode`` procedure fails.
```

Comment:
I would make this a caution/warning.

Reply:
I would elaborate here, adding another sentence or two, something like:
once a node is banned, the only way forward is to remove or replace it; you won't be able to perform other topology operations (decommission, bootstrap) until a banned node is removed from the cluster or replaced.

Reply:
Also similar section needs to be added to replace procedure --- the replaced node is permanently banned at the beginning! (Think of replace as removenode combined with bootstrap, happening at the same time)